### PR TITLE
Create comp

### PR DESCRIPTION
**Pull request by minsungpark89**
Thank you for pushing your code!

What (if any) features are you implementing?
- Features implemented was to add elements within the HTML and rules in CSS to build out the basic comp.  

What (if anything) did you refactor?
- WIP, elements will need to be renamed possibly.

Were there any issues that arose?
- CSS rules will need to be changed as problems may arise in future iterations.

Is there anything that you need from your teammate?
- Peer-review and also plan next steps for iteration 1!

Any other comments, questions, or concerns?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      <section class="new-activity-section">
+        <!-- create header for left container -->
+        <h2 class="new-activity">New Activity</h2>
+        <div class="form-container">
+          <!-- create body of left container -->
+          <section class="new-act-section"></section>
+          <!-- create 3 children buttons for study, meditate, exercise inside left container -->
+        <form class="form" id="form">
+          <section class="buttons">
+            <button type="button" class="form-button study-btn" id="Study">
```
Comment (najinl) [inline index.html:21]: In the buttons section, lets make sure that we delete or make kabob case the class names if we don't need to utilize both parts of the name in our CSS(i.e. before the space and after).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+              <img src="assets/study.svg" alt="light-blub" class="study">Study</button>
```
Comment (najinl) [inline index.html:22]: light "blub" ? I like your naming convention :)

**Review by Logandv3 — APPROVED** [review]
We may need to refactor a little for clarity.  We will need to delete comments at some point, but I like them for now to help see what is happening.  It looks good so far!  Great job getting this to look like the comp!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+          </section>
+          <!-- create input field for minutes and seconds -->
+            <section class="time">
+              <section class="minute">
+                <label for="minutes">Minutes</label>
+                <input type="number" name="minutes" id="minutes" class="input-field">
+              </section>
+              <section class="second">
+                <label for="seconds">Seconds</label>
+                <input type="number" name="seconds" id="seconds" class="input-field">
```
Comment (najinl) [inline index.html:41]: Do we need the name populated for these "time" nodes, the DOM text is being populated by the <label>

**Review by najinl — COMMENTED** [review]
These are all small things, everything looks great! Let's refactor together as we work on iteration 1.

**Pull request merged**